frontend/packages/knative-plugin/src/components/traffic-splitting/TrafficModalRevisionsDropdownField.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { set, cloneDeep } from 'lodash';
+import { useField } from 'formik';
+import { DropdownField } from '@console/shared';
+import { RevisionItems } from '../../utils/traffic-splitting-utils';
+
+type TrafficModalRevisionsDropdownFieldProps = {
+  revisionItems: RevisionItems;
+  name: string;
+  title: string;
+};
+
+const TrafficModalRevisionsDropdownField: React.FC<TrafficModalRevisionsDropdownFieldProps> = ({
+  revisionItems,
+  name,
+  title,
+}) => {
+  const [field] = useField(name);
+  const dropdownItems =
+    !field.value || Object.values(revisionItems).includes(field.value)
```

Comment:
nit: can we it as below
```suggestion
    Object.values(revisionItems).includes(field.value)
```

Reply:
this will allow empty value to be shown in dropdown resulting in a field with blank title

Reply:
am not sure m why will that be the case?